cypress/e2e/specialFeatures/atiAnalytics/assertions/featuresAnalysis.js

```diff
@@ -0,0 +1,43 @@
+import {
+  awaitATIComponentViewEvent,
+  interceptATIAnalyticsBeacons,
+  COMPONENTS,
+  awaitATIComponentClickEvent,
+} from '../helpers';
+
+const { FEATURES } = COMPONENTS;
+
+export const assertFeaturesAnalysisComponentView = () => {
+  it('should send a view event for the Features & Analysis component', () => {
+    cy.url().then(url => {
+      interceptATIAnalyticsBeacons();
+      cy.visit(url);
+
+      cy.get('[data-testid="features"]').scrollIntoView({ duration: 1000 });
+
+      awaitATIComponentViewEvent(FEATURES);
```

Comment:
For this (and all other assertions), should we assert that the request matches the data specified in the test suite e.g. pageIdentifer, contentType, applicationType? (similar to https://github.com/bbc/simorgh/blob/cypress-ati-tracking/cypress/e2e/specialFeatures/atiAnalytics/assertions/index.js#L18-L23)